build.rs

```diff
@@ -85,6 +86,12 @@ fn main() {
         _ => (),
     }
 
+    let musl_v1_2_3 = env::var("RUST_LIBC_UNSTABLE_MUSL_V1_2_3").is_ok();
+    println!("cargo:rerun-if-env-changed=RUST_LIBC_UNSTABLE_MUSL_V1_2_3");
+    // loongarch64 and ohos have already updated
+    if musl_v1_2_3 || target_os == "loongarch64" || target_env == "ohos" {
+        set_cfg("musl_v1_2_3");
+    }
     let linux_time_bits64 = env::var("RUST_LIBC_UNSTABLE_LINUX_TIME_BITS64").is_ok();
     println!("cargo:rerun-if-env-changed=RUST_LIBC_UNSTABLE_LINUX_TIME_BITS64");
     if linux_time_bits64 {
```

Comment:
If musl_v1_2_3 is enabled, should `linux_time_bits64` also be set for `__USE_TIME_BITS64` in UAPI? It's fine if you plan to do the needed time changes later, but it should be a FIXME.

Reply:
added a FIXME